### PR TITLE
Adtelligent bid adapter updated to v3.0

### PR DESCRIPTION
**Pull request by GeneGenie**
## Type of change
- [x] AdtelligentBidAdapter support for 3.0

## Description of change
Remove deprecated code 
Tests updated
Do not trigger same usersyncs across all auctions

- contact email of the adapter’s maintainer g.moroz@adtelligent.com
- [x] official adapter submission

### CODE REVIEW

File: modules/adtelligentBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+    const extMessage = serverResponse && serverResponse.ext && serverResponse.ext.message ? `: ${serverResponse.ext.message}` : '';
+    const errorMessage = `in response for ${bidderRequest.bidderCode} adapter ${extMessage}`;
+
+    utils.logError(errorMessage);
+
+    return bids;
+  }
+
+  serverResponse.bids.forEach(serverBid => {
+    const requestId = findIndex(bidderRequest.bids, (bidRequest) => {
```
Comment (jaiminpanchal27) [inline modules/adtelligentBidAdapter.js]: Any strong reason to use find-index from core-js. We have a white list of core-js modules here https://github.com/prebid/Prebid.js/blob/master/allowedModules.js#L2 
Can you update the logic ? 
Also, we have a custom eslint rule but not sure why it did not catch this import. I will look into this.

Comment (GeneGenie) [reply modules/adtelligentBidAdapter.js]: nope, just left from previous dev =)

Comment (GeneGenie) [reply modules/adtelligentBidAdapter.js]: so, can i use Array.prototype.findIndex ?

Comment (jaiminpanchal27) [reply modules/adtelligentBidAdapter.js]: @GeneGenie Array.prototype.findIndex does not support Internet explorer and hence the previous dev used core-js to support IE. I would suggest to use `find` from core-js

Here is the updated code which worked fine on my test page. Do test in on your end as well.

```
const request = find(bidderRequest.bids, (bidRequest) => {
      return bidRequest.bidId === serverBid.requestId;
    });

    if (serverBid.cpm !== 0 && request !== undefined) {
      const bid = createBid(serverBid, getMediaType(request));

      bids.push(bid);
    }
```

Comment (GeneGenie) [reply modules/adtelligentBidAdapter.js]: fixed

**Pull request merged**